Gitlab/two-stage-pipeline-template/cookiecutter.json

```diff
@@ -0,0 +1,17 @@
+{
+  "outputDir": "aws-sam-pipeline",
+  "main_git_branch": "",
+  "sam_template": "",
```

Comment:
I think we should have a default value here set to `template.yaml`. Same thing should also be applied to Jenkins and GitHub-Actions templates. SAM consider `template.yaml` to be the default SAM template file.